src/Directions.tsx

```diff
@@ -0,0 +1,103 @@
+import React from 'react'
+
+import { MapContext } from './Map'
+
+import { createCoordinate, NumberTuple } from './utils'
+
+type annotationOptions = {
+  origin?: mapkit.MarkerAnnotationConstructorOptions
+  destination?: mapkit.MarkerAnnotationConstructorOptions
+}
+
+type DirectionsProps = {
+  origin: NumberTuple
+  destination: NumberTuple
+  transportType?: 'Automobile' | 'Walking'
+  style?: mapkit.StyleConstructorOptions
+  options?: mapkit.DirectionsConstructorOptions
+  showAnnotations?: boolean
+  annotationOptions?: annotationOptions
+  routeDetails?: (arg0: {
```

Comment:
I wonder if we should change this prop name to something like `onRouteUpdate` to be more consistent with similar functions in react.

Reply:
I agree with this, and have changed this within my next push